addons/hr_timesheet/report/timesheets_analysis_report.py

```diff
@@ -25,6 +25,7 @@ class TimesheetsAnalysisReport(models.Model):
     unit_amount = fields.Float("Time Spent", readonly=True)
     partner_id = fields.Many2one('res.partner', string="Partner", readonly=True)
     milestone_id = fields.Many2one('project.milestone', related='task_id.milestone_id')
+    message_partner_ids = fields.Many2many('res.partner', readonly=True)
```

Comment:
How do we get the records in that field if we do not alter the query for the report? 🧐

Reply:
hmm, true, TBF I just added the field because I edited the ir.rule.

I think it's not worth altering the query just for this change?
Options:
1) I alter the query (potentially slowing it down quite a bit just to go search for the followers)
2) revert the change of ir.rule for the report

IMHO I would lean on option 2) to keep the diff smaller, the ir.rule for the report isn't called all the time (only when accessing the report).

What do you think?

Reply:
Yeah I agree we could go to the option 2.

Or option 3, we could implement the compute/search in that model as well. 🤷🏼‍♂️